modules/auxiliary/scanner/ssh/ssh_login.rb

```diff
@@ -35,7 +35,8 @@ def initialize
 
     register_options(
       [
-        Opt::RPORT(22)
+        Opt::RPORT(22),
+        OptBool.new('NO_SHELL', [ false, 'Do not create a shell on sucessful login.', false]),
```

Comment:
I believe you meant "successful." Also, there is mismatched whitespace around your brackets.